src/helpers/errors.ts

```diff
@@ -1,38 +1,50 @@
 import { HTTPError, NetworkError } from '@opensrp/server-service';
-import { toast } from 'react-toastify';
+import { toast, ToastOptions } from 'react-toastify';
 import { ACCESS_DENIED, NETWORK_ERROR, USER_HAS_NO_VALID_ASSIGNMENTS } from '../configs/lang';
 import { growl } from './utils';
 
 /** union of all ErrorTypes that we are working with */
 type ServiceError = HTTPError | Error | NetworkError;
 
+/** autoClose types */
+type AutoClose = 5000 | false;
```

Comment:
Should we fix this 5000ms? I think it should be flexible for different use cases.

Reply:
I was thinking in terms of not accepting any value. We define what values are acceptable. So that we can avoid a developer passing 1000 for instance which will make the toast very short or perhaps a value of 10000 which will make it very long. We can add more acceptable values in the type if need be

Reply:
Though 5000 is the [default value](https://fkhadra.github.io/react-toastify/api/toast-container) used by `react-toastify`

Reply:
My view is we don't need to fix it but to have 5000 as the default just the way you have put it.

Reply:
@kelvin-muchiri I will have to agree with Erick on this. i do not think we should have  a parameter that is in essence a constant.Also please bear in mind that we have an env that controls the auto close period across the whole app: `TOAST_AUTO_CLOSE_DELAY`.